gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/SettingsCategoryPanel.html

```diff
@@ -67,7 +67,10 @@ <h2 class="accordion-header">
 						<tbody>
 							<tr wicket:id="categoriesView" class="gb-category-row">
 								<td>
-									<a href="javascript:void(0);" class="gb-category-sort-handle ui-sortable-handle"></a>
+									<a href="javascript:void(0);"
+											class="gb-category-sort-handle ui-sortable-handle"
+											wicket:message="title:sortgradeitems.sort_category_label,aria-label:sortgradeitems.sort_category_label">
+									</a>
```

Comment:
<button type="button" class="btn btn-link">Link</button> ?

Reply:
these things are not really buttons or links, tbh. Just drag handles. The only reason they are links are just so they are in the natural tab order and focusable.